drivers/gles3/shaders/tonemap.glsl

```diff
@@ -156,6 +156,10 @@ vec3 tonemap_aces(vec3 color, float white) {
 }
 
 vec3 tonemap_reinhard(vec3 color, float white) {
+	// Ensure color values are pozitive
+	// They can be negative in the case of negative lights, which leads to undesired behaviour
```

Comment:
```suggestion
	// Ensure color values are positive.
	// They can be negative in the case of negative lights, which leads to undesired behavior.
```

Reply:
Ah, the "pozitive" typo was my bad and the behavior vs behaviour thing often confuses me :sweat_smile: